hera_qm/xrfi.py

```diff
@@ -229,7 +229,7 @@ def robust_divide(num, den):
 # Functions for flagging auto-correlations
 # ############################################################################
 
-def detrend_dpss_filt(data, noise, freqs, filter_centers, filter_half_widths, wgts=None,
+def dpss_flagger(data, noise, freqs, filter_centers, filter_half_widths, flags=None,
```

Comment:
Update the docstring returns to reflect that it returns flags not weights

Reply:
Updated this